doc/ChangeLog

```diff
@@ -80,7 +80,10 @@ version 26-git (????-??-??):
     part of system [chilan]
   - improvement: added 'phpui.send_message_confirm_threshold' configuration variable which control
     how many potential message recipients trigger confirmation dialog ('50' by default) [milek]
-
+  - enhancement: setting: rt.allow_modify_resolved_tickets_newer_than, allow to stop reopening
```

Comment:
Dopisz jaka wartość domyślna tutaj - spójrz jak dla innych wpisów jest to robione.